### ISSUE TITLE
next.config.js: TypeError: Cannot read property 'push' of undefined

### ISSUE THREAD

**Issue by SeanDez** [zeit/next.js#9349]
# Bug report

## Describe the bug

```
// terminal
owner@G700:~/PhpstormProjects/shopify/buyUsed$ next
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
TypeError: Cannot read property 'push' of undefined
```

```
// next.config.js
module.exports = withCSS({
  webpack: config => {
    const env = { API_KEY: apiKey };
    config.plugins.push(new webpack.DefinePlugin(env));
  
    config.node.push({fs: "empty"});
    
    return config;
  }
});
```

Looks like the node object either isn't there or I'm not accessing it in the correct way.

## A clear and concise description of what the bug is.

In a non-next.js project I have this:

```
// webpack.config.js
, node : {
    fs: "empty" // for dotenv to work correctly
  }
```

I am trying to get my next.config to replicate this setting.

## To Reproduce

Copy the same line to your next config file and then run `next` in terminal to see the error in console.

## Expected behavior

The dev server runs/watches without error.

## System information

- OS: Lubuntu 18.04
- Version of Next.js: 9.1.3

## Additional context

Shopify APP Cli is scaffolding this project.

**Comment by kachkaev** [comment]
This does not look like a bug in Next.js to me.

```config.node.push({fs: "empty"});``` would only work if `config.node` is an array. In your case, `config.node` is `undefined`, so you can't call `push` for this nothingness. You can try `console.log(config)` or `console.log(config.node)` to see that.

 I'm not sure I understand the value of setting `node → fs`, but syntactically, this would be `config.node = { fs: "empty" }`.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
